src/app/components/home/search/search.component.html

```diff
@@ -41,5 +41,5 @@
       }
     </label>
   </form>
-  <app-product-results [products]="products" />
+  <app-product-results [products]="allProducts$ | async" />
```

Comment:
Could wrap this in an @if instead of doing `@Input() products: ProductInterface[] | null | undefined = [];`